tests/cpp.test

```diff
@@ -178,6 +178,8 @@
 30313  sp_brace_brace-r.cfg                 cpp/sp_brace_brace.cpp 
 30314  sp_brace_brace-f.cfg                 cpp/sp_brace_brace.cpp
 
+30400  attribute_specifier_sequence.cfg     cpp/attribute_specifier_sequence.cpp
```

Comment:
Does this test fail without your changes? Should the config maybe have some options that would cause (mal)formatting if attributes are not detected?